src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -655,7 +656,7 @@ public void clearCache() {
      * @param args the arguments of message in respective properties file.
      * @return a string containing extracted localized message
      */
-    private String getLocalizedMessage(String messageKey, Object... args) {
+    public String getLocalizedMessage(String messageKey, Object... args) {
```

Comment:
Let's not open this method to users.
All public methods are contract with customers,  we should minimize contractual obligations.

I am fine to copy this method to test clas to get localed message